src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -246,6 +260,10 @@ private void setup(OptionSet options, Properties properties) {
 		if ( options != null && options.has("replay")) {
 			this.replayMode = true;
 		}
+
+		boolean outputBinlogPosition = fetchBooleanOption("output_binlog_position", options, properties, false);
+		boolean outputCommitInfo = fetchBooleanOption("output_commit_info", options, properties, false);
```

Comment:
should I change the flag to accept arguments (true/false?) or rename it `no_output_commit_info`?